datamonster_api/lib/datamonster.py

```diff
@@ -247,3 +256,219 @@ def _avro_to_df(self, avro_buffer):
         del df['lowerDate']
 
         return df
+
+    ##############################################
+    #           Dimensions methods
+    ##############################################
+
+    SPLITS_PAGESIZE = None      # or = 500
+
+
+    def get_dimensions_for_datasource(self, datasource, filters=None,
```

Comment:
do we really want a private variable in a public api?

Reply:
Not sure what precept this might violate, so I'd have to reply, Why ever not?
In this case, it's halfway between no variable and somehow exposing pagesize to users. It allowed exploration. I'm not attached to it, so it's gone.